mr/operator_executor.cc

```diff
@@ -47,4 +63,39 @@ void OperatorExecutor::SetMetaData(const pb::Input::FileSpec& fs, RawContext* co
   }
 }
 
+util::VarzValue::Map OperatorExecutor::GetStats() {
+  util::VarzValue::Map res;
+
+  LOG(INFO) << "MapperExecutor::GetStats";
+
+  auto start = base::GetMonotonicMicrosFast();
+  absl::flat_hash_map<string, uint64_t> input_read;
+
+  // TODO: Implement finish-latency-usec, a variable that tells us how much time, on average,
```

Comment:
You might call `GetStatsInternal` here which can be a virtual function that is overriden by descendants to provide additional operator specific statistics.

Reply:
I prefer to just implement it for both later on. If there will be a need for variables that are really unique, I will do as you say.